.prettierignore

```diff
@@ -8,4 +8,5 @@ golden/
 src/website/src/releases/final-template/auto-generated-routes.ts
 src/website/src/sitemap.xml
 src/clr-base/utils/css-vars/css-vars-ponyfill-fork.ts
-src/clr-base/**/*.css.ts
\ No newline at end of file
+src/clr-base/**/*.css.ts
```

Comment:
I'm worried about the number of files that we're generating in our src directory, can't we be putting them into `dist` or somewhere else?

Reply:
TypeScript won't allow the imports to be outside the root source directory, I could put them in a dist inside the base src `/src/clr-base/dist/styles/` then in the component you would import

`import { styles } from './../dist/styles/button.element.css';`

instead of 

`import { styles } from './button.element.css';`

The project will only generate one file per component. Everything else will live in clr-ui.

Reply:
I'll work on this some more see if I can get some path mapping to help with this.